shotover/src/runner.rs

```diff
@@ -182,7 +182,7 @@ impl Shotover {
         let mut runtime_builder = runtime::Builder::new_multi_thread();
         runtime_builder
             .enable_all()
-            .thread_name("Shotover-Proxy-Thread")
+            .thread_name("shotover-worker")
```

Comment:
Does this have any other side effects?

Reply:
Afaik it just changes the process name in htop and similar

Reply:
That could cause scripts/other tools that rely on the name being "Shotover-Proxy-Thread" to break.

Reply:
hmm fair enough, I'll pull it into its own PR so it can be considered seperately